apertium-uzb.uzb.lexc

```diff
@@ -35900,6 +35932,7 @@ erkatoy:erkatoy N1 ; ! 0.9999965658440407
 erka:erka N1 ; ! 0.9995437488829819
 erkinlik:erkinlik N1 ; ! 0.999999999995661
 erkin:erkin A1 ; ! "free"
+erkin:erkin ADV ; ! "freely"
```

Comment:
This isn't covered by `<adj><advl>` readings?

Reply:
Apparently, it is. So it is not really necessary to add them separately?.

Reply:
I guess I got a grasp of this now (looking at examples from kaz-tat), fixed it.